test/dlc_tests/ec2/pytorch/inference/test_pytorch_inference.py

```diff
@@ -87,8 +87,8 @@ def test_ec2_pytorch_inference_eia_gpu(pytorch_inference_eia, ec2_connection, re
 @pytest.mark.parametrize("ec2_instance_type", PT_EC2_GPU_INSTANCE_TYPE, indirect=True)
 def test_pytorch_inference_torchaudio_gpu(pytorch_inference, ec2_connection, gpu_only, ec2_instance_type):
     _, image_framework_version = get_framework_and_version_from_tag(pytorch_inference)
-    if Version(image_framework_version) not in SpecifierSet(">=1.9,!=1.10.*"):
-        pytest.skip("torchaudio is installed in PT 1.9 and above, except PT 1.10.*")
+    if Version(image_framework_version) not in SpecifierSet("!=1.9.*,!=1.10.*"):
+        pytest.skip("torchaudio is not supported in PT 1.9 and 1.10")
```

Comment:
I do see torchaudio installed in PT1.9 dockerfiles?

Reply:
@waytrue17 do you know why we are skipping PT 1.9 torchaudio?

Reply:
Because it didn't really work.